opal/tests/test_core_fields.py

```diff
@@ -1,6 +1,7 @@
 """
 Unittests for the opal.core.fields module
```

Comment:
You know, I think we may not have any explicit unittests for FT in this file, which I would really expect !

Reply:
I tested but the True and False cases in the methods


ie This is the case sensitive miss for a lookup list
https://github.com/openhealthcare/opal/pull/1534/files/96b2cfad409cc07583ceecdab4ed63be73371110#diff-7984144a820e90113b1c914866eb396aR117

This is a case sensitive miss for a synonym
https://github.com/openhealthcare/opal/pull/1534/files/96b2cfad409cc07583ceecdab4ed63be73371110#diff-7984144a820e90113b1c914866eb396aR143